### ISSUE TITLE
No working app 

### ISSUE THREAD

**Issue by Rakhimjon** [abhinav272/PagingSample#1]
![screenshot from 2018-10-02 17-48-34](https://user-images.githubusercontent.com/22024327/46349503-a3ff0680-c66b-11e8-8ded-b0ab35d1fb98.png)

**Comment by abhinav272** [comment]
@Rakhimjon Strange but mine works fine.
Have you changed something?

U can also try invalidating and restarting

**Comment by Rakhimjon** [comment]
Yes I have changing   minSdkVersion 16 and  news api key owner api key seting

**Comment by abhinav272** [comment]
@Rakhimjon can you try running app without changing anything except news API key.
Also, can you paste the complete stacktrace
Check if the issue is because I have mentioned LayoutManager tag in XML(relative layout), remove that and set in java code, maybe that's the issue
